src/transformers/convert_slow_tokenizer.py

```diff
@@ -36,6 +35,11 @@ class SentencePieceExtractor:
 
     def __init__(self, model: str):
         # Get SentencePiece
+        try:
```

Comment:
Yes in this case I actually wanted to add a more specific error message mentioning the possibility to use a tokenizer.json file if available instead of installing SP